### PR TITLE
[CMSP-268] Update WordPress Multisite Search & Replace functionality documentation

### PR DESCRIPTION
**Pull request by jazzsequence**
## Summary

**[Search and Replace](https://docs.pantheon.io/guides/multisite/search-replace/)** - Updates configuration instructions and notes to correspond with upcoming changes.

### Dependencies and Timing

- [x] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [x] After date: June 30(?)

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/old-path/` => `/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by jazzsequence** [comment]
[Preview](https://pr-8488-documentation.appa.pantheon.site/guides/multisite/search-replace/)

**Review by whitneymeredith — APPROVED** [review]
LGTM

**Comment by jazzsequence** [comment]
@whitneymeredith confirm with @scottbuscemi for merge timing since this stuff isn't out yet. :)

File: source/content/guides/multisite/06-search-replace.md [hunk 1]
```diff
@@ -18,33 +18,41 @@ editpath: multisite/07-search-replace.md
...
-You must have the following to enable search and replace:
+You must have the following to enable Search and Replace:
 
 - A [WordPress Multisite](/guides/multisite)
 
-## Enable Search and Replace
+## Search and Replace Configuration
 
-You can enable search and replace between environments on a WPMS site.
+Search and Replace will automatically work for you without any configuration if you have a new `_subdirectory multisite_`. However, you can modify the behavior of Multisite Search and Replace by editing the `search_replace` value.
```
Comment (scottbuscemi) [inline source/content/guides/multisite/06-search-replace.md:37]: @jazzsequence Could we remove this paragraph for this PR? Ideally, we merge the new pantheon.yml changes for EA participants then put this paragraph in a new PR

Comment (jazzsequence) [reply source/content/guides/multisite/06-search-replace.md:37]: Remove wholesale or remove and revert to the original content?

Comment (scottbuscemi) [reply source/content/guides/multisite/06-search-replace.md:37]: Revert to the original content :)

Comment (jazzsequence) [reply source/content/guides/multisite/06-search-replace.md:37]:
```suggestion
You can enable search and replace between environments on a WPMS site.
```

Comment (jazzsequence) [reply source/content/guides/multisite/06-search-replace.md]: Note for future selves: At some point in the future when we've implemented, we will want to switch this _back_.

File: source/content/guides/multisite/06-search-replace.md [hunk 2]
```diff
@@ -18,41 +18,44 @@ editpath: multisite/07-search-replace.md
...
 
 </Alert>
 
-This section provides information on how to configure automatic platform search and replace for URLs. This feature can be used with both subdirectory and subdomain WordPress Multisites.
+This section provides information on how to configure automatic platform Search and Replace for URLs. This feature can be used with both subdirectory and subdomain WordPress Multisites.
 
-Search and replace runs on the platform as part of creating an environment, deploying to Test or Live, and when cloning content between environments. There is currently a limit of 25 sites that can be configured for search and replace. We expect this number to increase before General Availability.
+Search and Replace runs on the platform as part of creating an environment, deploying to Test or Live, and when cloning content between environments. There is currently a limit of 25 sites that can be configured for search and replace. We expect this number to increase before General Availability.
```
Comment (pwtyler) [inline source/content/guides/multisite/06-search-replace.md:27]: > There is currently a limit of 25 sites that can be configured for search and replace.

This belongs only under `custom` search-replace, there is no limit when using "standard" (`true`) subdirectory search-replace and/or conversion on pull from live.

File: source/content/guides/multisite/06-search-replace.md [hunk 3]
```diff
@@ -103,27 +106,29 @@ Environments that need to be replaced are defined and paired in the `sites.yml`
...
+
+1. Set the `SUBDOMAIN_INSTALL` value to conditional:
 
     ```php:title=wp-config.php
     if ( ! empty( $_ENV['PANTHEON_ENVIRONMENT'] ) && $_ENV['PANTHEON_ENVIRONMENT'] ==  'live' ) {
       define( 'SUBDOMAIN_INSTALL', true );
     }
     ```
 
-1. Set `convert_to_subdirectory: true` in the `sites.yml` file.
```
Comment (pwtyler) [inline source/content/guides/multisite/06-search-replace.md:120]: Do we want a note/alert in the docs to the effect of "this used to work and no longer does", or is that more confusing?

Comment (scottbuscemi) [reply source/content/guides/multisite/06-search-replace.md:120]: Naw. I'll communicate to the EA folks.

File: source/content/guides/multisite/06-search-replace.md (hunk 2 above)
Comment (jazzsequence) [reply source/content/guides/multisite/06-search-replace.md:27]: moved to line 49

**Comment by jazzsequence** [comment]
@whitneymeredith this is good to merge!
cc @pantheon-systems/docs-admins

**Pull request merged**